spec/integration/users_show_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'user#Show', type: :feature do
+  describe 'User' do
+    before(:each) do
+      @mohammed = User.create!(
+        name: 'mohammed',
+        photo: 'msMod.jpeg',
+        bio: 'Lorem ipsum dolor sit amet',
+        posts_counter: 0
+      )
+
+      @tom = User.create!(
+        name: 'tom',
+        photo: 'msMod.jpeg',
+        bio: 'Lorem ipsum dolor sit amet',
+        posts_counter: 0
+      )
+
+      visit root_path
+
+      @post1 = mohammed.posts.create!(title: 'Ruby and Rails 1', text: 'this is first post about Ruby and Rails! 1',
+                                      comments_counter: 0, likes_counter: 0)
+      @post2 = mohammed.posts.create!(title: 'Ruby and Rails 2', text: 'this is first post about Ruby and Rails! 2',
+                                      comments_counter: 0, likes_counter: 0)
+      @post3 = mohammed.posts.create!(title: 'Ruby and Rails 3', text: 'this is first post about Ruby and Rails! 3',
+                                      comments_counter: 0, likes_counter: 0)
+
+      visit user_path(@mohammed.id)
+    end
+    it "show user's profile picture" do
+      all('img').each do |i|
+        expect(i[:src]).to eq('msMod.png')
+      end
```

Comment:
- [ ] As explained earlier, please ensure that you are checking for a photo url that is present. At the moment, `msMod.png` is not.